Source/WebKit/NetworkProcess/mac/NetworkConnectionToWebProcessMac.mm

```diff
@@ -47,6 +47,14 @@
     RELEASE_LOG(Process, "NetworkConnectionToWebProcess::updateActivePages");
 #if USE(APPLE_INTERNAL_SDK)
     auto asn = adoptCF(_LSCopyLSASNForAuditToken(kLSDefaultSessionID, auditToken.auditToken()));
+    if (!asn) {
+#if ENABLE(LAUNCHSERVICES_SANDBOX_EXTENSION_BLOCKING)
+        // In this case, the WebContent process has not been checked in with Launch Services yet.
+        // We store the display name, and set it after checkin has completed.
+        m_pendingDisplayName = overrideDisplayName;
```

Comment:
The `overrideDisplayName` parameter for this function can be a `String&&` and you can use `WTFMove()` here.